app/controllers/collections_controller.rb

```diff
@@ -46,11 +49,14 @@ def destroy
 
   private
 
-    def set_collection
-      @collection = Collection.find_by uid: params[:id]
-    end
+  def set_collection
+    @collection = Collection.find_by uid: params[:id]
+  end
 
-    def collection_params
-      params.require(:collection).permit(:uid, :title, :description, :url, :image_url, :vendor_id, :vendor_identifier)
-    end
+  def collection_params
+    params.require(:collection).permit(:uid, :title,
+                                       :description, :url,
```

Comment:
One per line is much easier to read - the second one on each line is at a different offset with two.